public/app/plugins/datasource/loki/modifyQuery.test.ts

```diff
@@ -233,3 +239,39 @@ describe('addLabelFormatToQuery', () => {
     );
   });
 });
+
+describe('removeCommentsFromQuery', () => {
+  it('strips comments in log queries', () => {
```

Comment:
Have you considered using `it.each()` for some of these test scenarios?

Reply:
You are right and `it.each` would be better for future debugging and identifying what actually failed. 

Updated in: https://github.com/grafana/grafana/pull/53254/commits/69e461b8f9b858915086fc577923305b2eb26737